homeassistant/components/xiaomi_miio/humidifier.py

```diff
@@ -197,24 +177,24 @@ def __init__(self, name, device, entry, unique_id, coordinator):
         """Initialize the plug switch."""
         super().__init__(name, device, entry, unique_id, coordinator)
         if self._model in [MODEL_AIRHUMIDIFIER_CA1, MODEL_AIRHUMIDIFIER_CB1]:
-            self._available_modes = AVAILABLE_MODES_CA1_CB1
-            self._min_humidity = 30
-            self._max_humidity = 80
+            self._attr_available_modes = AVAILABLE_MODES_CA1_CB1
+            self._attr_min_humidity = 30
+            self._attr_max_humidity = 80
```

Comment:
As these seem to be the same for all models, setting them in a single place would simplify the code.

Reply:
Yes, good point.